packages/docs/pages/padding.tsx

```diff
@@ -1,59 +1,74 @@
-              <NextLink href="/spacing">Spacing</NextLink> type.
-            </Text>
-
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <Box padding={{ mobile: 'none', tablet: 'medium', desktop: 'xLarge' }} backgroundColor="secondary20">
-                Example
-              </Box>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => <PaddingPropTable />,
-    },
-  ];
-
   return (
     <>
       <H1>Padding</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          BigDesign provides the <Code primary>padding</Code> prop which can be used in some of our components.
+        </Text>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <Fragment key="basic">
+                  <Text>
+                    The following example showcases the simplest way to use the <Code primary>padding</Code> prop by
+                    giving it a <NextLink href="/spacing">Spacing</NextLink> value.
+                  </Text>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Box padding="medium" backgroundColor="secondary20">
+                      Example
+                    </Box>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+            {
+              id: 'responsive-breakpoints',
+              title: 'Responsive breakpoints',
+              render: () => (
+                <Fragment key="responsive-breakpoints">
+                  <Text>
+                    You can also specify paddings with an object to handle different paddings with different{' '}
+                    <NextLink href="/breakpoints">breakpoints</NextLink>. All values must be of{' '}
+                    <NextLink href="/spacing">Spacing</NextLink> type.
+                  </Text>
+
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Box
+                      padding={{ mobile: 'none', tablet: 'medium', desktop: 'xLarge' }}
+                      backgroundColor="secondary20"
+                    >
+                      Example
+                    </Box>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <PaddingPropTable renderPanel={false} />
+      </Panel>
     </>
```

Comment:
Note:
I was thinking of adding the following in the Guideline section:
All values must be of{' '}Spacing type.

However, I feel that the <PaddingPropTable/> shows the Spacing as the type, so might not be necessary to mention it in the Guideline section.

Other than that, I think there is nothing important that I can think of 🤔 ... Does someone has a suggestion of what could be added in this section?